packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.tsx

```diff
@@ -154,6 +151,9 @@ const MultiInputDateRangeField = React.forwardRef(function MultiInputDateRangeFi
   const separatorProps = useSlotProps({
     elementType: Separator,
     externalSlotProps: slotProps?.separator,
+    additionalProps: {
```

Comment:
I also applied the new prop to the multi input range fields because IMHO it makes the DX more consistent (even though people could use `slotProps.fieldSeparator.children` to achieve the same behavior.
Note that the old DX still works.